packages/fast-components-react-msft/src/paragraph/paragraph.tsx

```diff
@@ -0,0 +1,68 @@
+import * as React from "react";
+import * as ReactDOM from "react-dom";
+import { get } from "lodash-es";
+import { Foundation, HandledProps, TypeLevel, TypographyTag } from "@microsoft/fast-components-react-base";
+import {
+    IParagraphHandledProps,
+    IParagraphManagedClasses,
+    IParagraphUnhandledProps,
+    ParagraphLevel
+} from "./paragraph.props";
+import Typography from "../typography";
+import { IManagedClasses, IParagraphClassNameContract } from "@microsoft/fast-components-class-name-contracts-msft";
+
+/* tslint:disable-next-line */
+class Paragraph extends Foundation<IParagraphHandledProps & IManagedClasses<IParagraphClassNameContract>, React.HTMLAttributes<HTMLParagraphElement>, {}> {
+    public static defaultProps: Partial<IParagraphHandledProps> = {
+        level: ParagraphLevel._3
+    };
+
+    protected handledProps: HandledProps<IParagraphHandledProps & IManagedClasses<IParagraphClassNameContract>> = {
+        level: void 0,
```

Comment:
add children

Reply:
Let's sync with @nicholasrice on this as we are inconsistent on this and we should land and file an issue if needed to align things.

Reply:
Synced with @nicholasrice, we'll add children in foundation